pkgs/top-level/all-packages.nix

```diff
@@ -27405,7 +27405,7 @@ with pkgs;
   bonzomatic = callPackage ../applications/editors/bonzomatic { };
 
   bottles = callPackage ../applications/misc/bottles {
-    wine = wineWowPackages.minimal;
+    wine = null;
```

Comment:
Why do we overwrite this to null?

Reply:
You can find the reasoning in the thread. I'll revert this if you don't agree.

Reply:
If we want to disable features we should not remove a particular packages but use a boolean which is way nicer.

Reply:
There are many versions of Wine a user may want, a boolean would force a specific one.
There is a discussion about it above (in "Conversation", not visible in "File changed" for some reason...)
CTRL+F `Can this be kept`.